index.html

```diff
@@ -109,6 +111,8 @@
   <a href="#setup">below</a>). They are also required to abide by
   Software Carpentry's
   <a href="{{site.swc_site}}/conduct.html">Code of Conduct</a>.
+  <strong>As mention before, you don't need to have any previous knowledge of the tools that will
+    be presented at the workshop.</strong>
```

Comment:
I would take this out - the mention under 'Who' is enough. Once that's done, please go ahead and merge.